cypress/integration/multi-pdf/navigations.ts

```diff
@@ -22,8 +24,8 @@ describe('Multi PDF navigation', () => {
     cy.findByRole('radio', { name: 'Paginated' }).should('not.be.checked');
     cy.findByRole('radio', { name: 'Scrolling' }).should('be.checked');
     cy.findByRole('button', { name: 'Table of Contents' }).click();
-    cy.findByRole('menuitem', { name: 'Foreword' }).click();
-    cy.wait(5000);
+    cy.findByRole('listitem', { name: 'Foreword' }).click();
+    cy.wait('@pdf');
```

Comment:
So this @pdf refers to the line in `/support/commands.ts` How do we decide what is a command and what is a util?

Reply:
eg: Why is /util/pdfZoomTestHelper not an additional command, instead of being a util function?

Reply:
Another q for @Toxiapo

Reply:
I think both works. I would define a command for this if this was being used elsewhere, but this is only testing the zoom value specifically.